### PR TITLE
Websockets Drawing Example

### PR DESCRIPTION
**Pull request by Waidhoferj**
Added simple collaborative drawing example to a new `examples` folder.

### CODE REVIEW

**Comment by davidbrochart** [comment]
Don't you want to use https://github.com/y-crdt/ypy-websocket for that?

**Comment by Waidhoferj** [comment]
@davidbrochart that was my original idea, but I ran into an error that I couldn't debug. We could figure out the issue and replace the websocket logic.

The error involved a buffer read that expected 2 bytes and received 0.

**Comment by Waidhoferj** [comment]
Sounds good, I'll create a minimal example and post an issue to the `ypy-websocket` repository.

File: examples/websocket-provider/client.py [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        _thread = threading.Thread(target=between_callback)
+        _thread.start()
+
+        
+
+
+    def send_updates(self, txn_event: Y.AfterTransactionEvent):
+        update = txn_event.get_update()
+        if update != b'\x00\x00':
+            self.send_q.put_nowait(update)
```
Comment (Waidhoferj) [inline examples/websocket-provider/client.py:32]: @Horusiath I noticed that the `observe_after_transaction` is running in a tight loop with an empty update message when nothing has changed. Is this happening due read-only transaction operations in the `draw()` loop that don't change the state?

File: examples/websocket-provider/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Web Socket Example
+
+Shows how you can use the Python WebSocket library to Ypy exchange updates between two users.
+
```
Comment (dmonad) [inline examples/websocket-provider/README.md:4]:
```suggestion
![drawing-demo](https://user-images.githubusercontent.com/5553757/189190756-21c2bc61-1816-488e-b2cd-bc910fece6d9.gif)
```
Add a drawing demo gif :)

john-drawing-demo

**Pull request merged**